### PR TITLE
Fake PR for discussion questions.

### PR DESCRIPTION
**Pull request by flamingbear**
This is an alternate way to get the bucket configuration information on
deploymen rather than have the distribution endpoint access the system_bucket
and read the buckets.json from s3.

### CODE REVIEW

File: packages/deployment/app/cumulus_api.template.yml [hunk 1]
```diff
@@ -178,6 +178,9 @@ Resources:
       Environment:
         Variables:
           stackName: {{../stackName}}
+          system_bucket: {{../parent.system_bucket}}
+          public_buckets: {{#collectBuckets ../parent.buckets "public"}}{{/collectBuckets}}
```
Comment (flamingbear) [inline packages/deployment/app/cumulus_api.template.yml]: So here I tried to use non-block helpers in handlebars, but couldn't get them to work.  Any ideas?
I was trying `{{{#collectBuckets ../parent.buckets "public"}}}` like the docs said, but no dice.

Comment (flamingbear) [reply packages/deployment/app/cumulus_api.template.yml]: Really I wanted to add these helpers to packages/api/config/distribution.yml, but was unable to get them to be interpreted in any manner.  Am I missing something fundamental about how kes works. .

something like ```  envs:
    EARTHDATA_BASE_URL: '{{parent.urs_url}}'
    EARTHDATA_CLIENT_ID: '{{EARTHDATA_CLIENT_ID}}'
    EARTHDATA_CLIENT_PASSWORD: '{{EARTHDATA_CLIENT_PASSWORD}}'
    public_buckets: '{{{#collectBuckets buckets "public"}}}'
    protected_buckets: '{{{#collectBuckets buckets "protected"}}}'
```  in this location https://github.com/nasa/cumulus/blob/master/packages/api/config/distribution.yml#L9-L12

Comment (Jkovarik) [inline packages/deployment/app/cumulus_api.template.yml]: You don't need {{/collectBuckets}}, you should definitely use the "{{{" invocation.   See "ToJson" .     What was the failure when you used this invocation?     What was the resultant template/failure?

Comment (Jkovarik) [reply packages/deployment/app/cumulus_api.template.yml]: Er, and yes, the handlebars helpers are only applied to the template object. 

See (in parseCF, in kes.js) 

```   const template = Handlebars.compile(t);
    return template(this.config);```

The config object interpolation of things like {{EARTHDATA_CLIENT_ID}} is being handled in kes's config.js (parseConfig).     As an aside, config.js is *not* overridable due to how the cli/index brings in the config before loading the kes class.

**Pull request closed without merging**